src/merkletree/merkletree.ts

```diff
@@ -0,0 +1,116 @@
+// intial proposition: https://github.com/Astraly-Labs/astraly-contracts/blob/main/tests/test_lottery_tickets.py
+
+const starknet = require("starknet");
+
+import {Grant} from "../Types";
```

Comment:
```suggestion
import {Grant} from "../types";
```
On évite les majuscules sur les noms de dossier/module